layouts/partials/fragments/list.html

```diff
@@ -77,16 +77,16 @@ <h5 class="col-12 col-md px-0">
                             </a>
                           </h5>
                         {{- end -}}
-                        {{- if and (not $self.Params.tiled) (ne $self.Params.display_date false) }}
+                        {{- if and (not $self.Params.tiled) (eq $self.Params.display_date false) }}
```

Comment:
Shouldn't that just be `{{- if and (not $self.Params.tiled) $self.Params.display_date }}`?